src/api/stats.ts

```diff
@@ -269,6 +274,45 @@ const getStatsForDetails = (
         .returns<CatalogStats_Details[]>();
 };
 
+export interface DefaultStatsWithDocument extends DefaultStats {
+    flow_document: any;
+}
+
+const getStatsForDashboard = (
+    tenant: string,
+    grain: Grains,
+    endDate: DateTime,
+    duration?: Duration,
+    entityType?: Entity
+) => {
+    const past = duration ? endDate.minus(duration) : endDate;
+
+    let query: string;
+    switch (entityType) {
+        case 'capture':
+            query = CAPTURE_QUERY;
+            break;
+        case 'materialization':
+            query = MATERIALIZATION_QUERY;
+            break;
+        case 'collection':
+            query = COLLECTION_QUERY;
+            break;
+        default:
+            query = DEFAULT_QUERY;
+    }
```

Comment:
Is there a specific future plan you have for supporting these queries? 

If this is just adding to handle it _in case_ I think we can comment out so reduce complexity for queries that aren't actually getting used.

Reply:
The addition of the data trends graph would duplicate a considerable amount of code for the data usage graph on the _Details_ page. Consequently, it was decided that the two graphs will share logic; this opens the door for `getStatsForDashboard` and `getStatsForDetails` to be consolidated into a single function. This branch is cherry picked from one which implements the initial plan for the dashboard.

**Tl;dr:** The highlighted code fragment can be commented out for now but will likely need to be reintroduced when the data trends graph is developed.

Reply:
Ahhhh - that makes sense. We can leave it in then.